utils/helpers.py

```diff
@@ -8,9 +8,7 @@ class YamlDumper(yaml.SafeDumper):
 
     def write_line_break(self, data=None):
         super().write_line_break(data)
-
-        if len(self.indents) == 1:
-            super().write_line_break()
```

Comment:
This code was removed so that the diff does not contain a new line between keys in fields.yml.

Prettier is in charge of formatting the file.